src/app/objective/objective.component.ts

```diff
@@ -37,10 +37,15 @@ export class ObjectiveComponent implements OnInit {
   @Output() onMoveBucket = new EventEmitter<[ImmutableObjective, ImmutableObjective, ImmutableBucket]>();
   @Output() onDelete = new EventEmitter<ImmutableObjective>();
   @Output() onChanged = new EventEmitter<[ImmutableObjective, ImmutableObjective]>();
+  @Input() sum?:number;
+  @Input() bucketAllocationLimit?:number;
+  @Output() cumiliativeSum= new EventEmitter<number>();
```

Comment:
I don't think we need these two things. We just need the cumulative sum (to display) and the bucket limit (to decide the colour).